public/styles/styles.css

```diff
@@ -388,3 +388,128 @@ ul.org-card {
     margin-top: 50px;
     padding-top: 10px;
 }
+
+/* add-experience css starts here */
```

Comment:
Please review the name convention you are using for classes. Its a mix of words all lowercase, camelcase or kebab case. For this project, we are using kebab case where there is a dash between each word. Please review this section and the add-experience ejs file.